index.js

```diff
@@ -65,8 +65,42 @@ function toggleColor(element) {
     } else {
         element.style.color = "green"
     }
-}
+} 
+
+header.addEventListener('click', function(event){
+    toggleColor(event.target)
+})
 
 /***** Deliverable 2 *****/
 
+const likeButton = document.querySelector('button.like-button')
+
+likeButton.addEventListener('click', function(event){
+
+    likes.textContent = `${traveler.likes + 1} Likes`
+})
+
+
 /***** Deliverable 3 *****/
+const form = document.querySelector('form#new-animal-sighting-form')
+
+form.addEventListener('submit', function(event){
+ event.preventDefault()
+
+    const inputFieldOne = event.target[0].value
+    const inputFieldTwo = event.target[1].value
+    const inputFieldThree = event.target[2].value
+    const inputFieldFour = event.target[3].value
+
+     const newAnimalSightingObject = {
+        species: inputFieldOne,
+        link: inputFieldTwo,
+        photo: inputFieldThree,
+        description: inputFieldFour,
+        id: 5,
+    }
+
+   traveler.animalSightings.push(newAnimalSightingObject)
+   renderAnimalSightingPost(newAnimalSightingObject)
+   form.reset()
+})
```

Comment:
Great job here! Again, I would create a separate function to be passed in as a callback function. I would make the variable names more descriptive than "inputFieldOne", and change it to something descriptive of what it is, which is a string of information about an animal `animalSpecies`.

Great job assembling an object of the data!

You are thinking ahead with line 104, but we will have a more dynamic way of persisting the data to the backend. stay tuned :)

Great job using the pre-existing function on line 105!

+++++++++ for resetting the form!! 👏 

My steps for solving this are as follows:

1. Find the sightingForm on the page and save it to a variable
2. add an event listener to the form element, listen for a `submit` event.
3. The submit event needs to be applied to an entire form element, not the submit button.
4. Create the `handleFormSubmission` callback function, which should be triggered when the form submits.

`handleFormSubmission` steps:

1. since we are submitting a form, we need to prevent the default action of the form, which is to make a post request and redirect to a specific URL. We do not want the page to refresh, we want to prevent that action and stay on the current page.
2. save all the data the user typed in from the form into descriptive variables.
3. assemble an object with key value pairs descriptive of the data they contain.
4. Pass in the animal data object to the `renderAnimalSightingPost` function

```
const sightingForm = document.querySelector(`form#new-animal-sighting-form`);

function handleFormSubmission(e){
    e.preventDefault();
    const species = e.target["species"].value;
    const videoLink = e.target["animal-link"].value;
    const photo = e.target["animal-photo"].value;
    const description = e.target["sighting-description"].value;
    const newData = {
        species: species,
        videoLink: videoLink,
        photo: photo,
        description: description
    }
    renderAnimalSightingPost(newData);
}

sightingForm.addEventListener("submit", handleFormSubmission)
```

With ES6 notation

```
const sightingForm = document.querySelector(`form#new-animal-sighting-form`);

const handleFormSubmission = e => {
    e.preventDefault();
    const species = e.target["species"].value;
    const videoLink = e.target["animal-link"].value;
    const photo = e.target["animal-photo"].value;
    const description = e.target["sighting-description"].value;
    const newData = {
        species,
        videoLink,
        photo,
        description
    }
    e.target.reset()
    renderAnimalSightingPost(newData);
}

sightingForm.addEventListener("submit", handleFormSubmission)
```